recipes/wg21-linear_algebra/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
v1 validation needs to be a bit less "modern", see https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/header_only/all/test_v1_package/conanfile.py

Reply:
Ah, thanks for the clarification I've made that amendment.